di-ipv-credential-issuer-stub/src/main/java/uk/gov/di/ipv/stub/cred/CredentialIssuer.java

```diff
@@ -45,7 +44,7 @@ private void initRoutes() {
         Spark.get("/authorize", authorizeHandler.doAuthorize);
         Spark.post("/authorize", authorizeHandler.generateResponse);
         Spark.post("/token", tokenHandler.issueAccessToken);
-        Spark.get("/credential", credentialHandler.getResource);
+        Spark.post("/credentials/issue", credentialHandler.getResource);
```

Comment:
You will need to update core-back parameter store values to use this new endpoint

Reply:
Yep, there’s another story to make the changes to core back. I’d hold off merging this till that’s ready to save breaking things.